tests/cpp_random_unit.cpp

```diff
@@ -21,6 +21,7 @@
 //
 
 #include <assert.h>
+#include <inttypes.h>
```

Comment:
Prefer native C++ header
```
#include <cinttypes>
```

Actually if you wanted you could

```
#include <cassert>
#include <cinttypes>
#include <cstdio>
#include <cstdlib>
#include <cstring>
#include <ctime>
```

Reply:
Ok. Done.